examples/mobile/ScheduleApp/js/day-indicator.js

```diff
@@ -0,0 +1,57 @@
+class DayControl extends HTMLElement {
+
+	connectedCallback() {
+		this.classList.add("day-control");
+	}
+}
+
+customElements.define("day-control", DayControl);
+
+class DayIndicator extends HTMLElement {
+
+	constructor() {
+		super();
+		this.daysLabels = ["S", "M", "T", "W", "T", "F", "S"];
+	}
+
+	connectedCallback() {
+		this.classList.add("day-indicator");
+		this.addDays();
+		this.addActiveDays();
+	}
+
+	get active() {
+		return this.getAttribute("active");
+	}
+
+	set active(value) {
+		this.setAttribute("active", value);
+	}
+
+	addDays() {
+		var i;
+
+		this.innerHTML = "";
+		this.days = [];
+		for (i = 0; i < 7; i++) {
+			this.days[i] = document.createElement("day-control");
+			this.days[i].innerText = this.daysLabels[i];
+			this.appendChild(this.days[i]);
+		}
+	}
+
+	parseActiveDays() {
+		return this.active ? this.active.split(" ").map((el) => +el) : [];
+	}
+
+	addActiveDays() {
+		this.parseActiveDays().forEach((el) => {
+			this.days[el].classList.add("day-control-active");
+		})
+
+
+	}
+}
+
+
+customElements.define("day-indicator", DayIndicator);
```

Comment:
Question)
Is it working correctly on the tizen without transpiler or polyfill?

Reply:
I verified it on Tizen 5.5 emulator. Works good.